lib/scoped/find.js

```diff
@@ -18,8 +23,22 @@ function find (type, api, objectsOrIds) {
       if (storedObjects.type === type) {
         return storedObjects
       } else {
-        throw new Error('Item not found')
+        throw handleIncorrectTypeError(type, storedObjects)
       }
     }
   })
 }
+
+function handleIncorrectTypeError (type, storedObject) {
+  var errorMessage = 'Object with type "' + type + '" and id "' + storedObject.id + '" is missing'
+  var incorrectType = new Error(errorMessage)
+  incorrectType.name = 'Not found'
+  incorrectType.status = 404
+  return incorrectType
+}
+
+function allObjectsAreNotFound (typedObjects) {
+  return typedObjects.every(function (typedObject) {
```

Comment:
yes, we try to match PouchDB’s browser support: https://pouchdb.com/learn.html#browser_support pretty much everything after IE 10 right now